webcompat/static/favicon/safari-pinned-tab.svg

```diff
@@ -0,0 +1 @@
+<svg version="1" xmlns="http://www.w3.org/2000/svg" width="666.667" height="666.667" viewBox="0 0 500.000000 500.000000"><path d="M0 250v250h500V0H0v250zM270.7 40.4c21.5 5.8 35.5 22.3 39.1 45.8l.9 5.8H192v-2.1c0-4.2 3.2-14.3 6.7-21.2C206.6 53 222 41.9 240 39c7.2-1.2 24.2-.4 30.7 1.4zM311 116v14H191v-28h120v14zm3 37v14H186v-28h128v14zm4.1 35.2c5.7 17.6 7.8 21.2 34.9 60.7 14.8 21.6 21.4 35.9 25.7 56.1 2.6 12.2 2.6 47.9 0 59.9-8.7 40.1-29 67.7-62.3 84.4-32.9 16.4-79.7 19.3-117 7.1-42.1-13.7-70-45.2-79.6-89.7-2.9-13.7-3.2-46.5-.5-59.7 4.8-24 11.8-37.8 37.1-73 12.7-17.8 19-29.5 23.7-44.4l3.3-10.6 65.9.2 65.9.3 2.9 8.7z"/><path d="M243.5 43.9c-21.4 2.7-35.5 13.7-43 33.4-1.4 3.7-2.5 7.4-2.5 8.2 0 1.3 7 1.5 53.5 1.5H305l-.6-2.8c-6-25.3-21.9-39-47.4-40.6-4.1-.2-10.2-.1-13.5.3zM197 116v9h108v-18H197v9zM192 153v9h117.1l-.3-8.8-.3-8.7-58.2-.3-58.3-.2v9zM184.6 192.7c-5.7 17.3-9.2 23.5-28.5 50.8-18.6 26-25.7 39.6-30.1 57.1-4.3 17.6-4.9 42.2-1.5 62 6.4 36.7 27.5 65.8 59.3 81.8 31.9 16.1 75.8 18.8 113.2 7.1 10-3.2 26-11.1 33.3-16.6 31.4-23.5 47.3-61 45.4-106.9-1.3-30.2-7.8-48.2-28.1-77.8-5-7.3-12-17.4-15.4-22.5-7.7-11.2-13.4-22.2-17.8-34.2l-3.2-9-61.9-.3-61.9-.2-2.8 8.7zm44.8 5.5c-1.3 4.3-19.6 61.5-30.5 95.6-6.1 18.9-10.9 34.6-10.6 34.8.2.3 1.7 0 3.3-.7 8.8-3.3 23.2-2.9 33.8 1.2l5 1.9 5.5-2.8c4.9-2.4 6.8-2.7 15.1-2.7 7.5 0 10.5.5 14 2.1l4.5 2 4.5-2c6.8-3 19.3-2.7 28.8.6 7.3 2.6 7.3 2.6 6.6.5-.4-1.2-9.8-31.5-20.9-67.2l-20.2-65 3-.3c1.7-.2 3.4.2 3.8 1 .5.7 11.5 33.4 24.4 72.6 15.5 46.9 23.5 72.5 23.3 74.5-.4 4.4-2.9 4.5-7.5.5-5.1-4.6-7.7-6.1-14.8-8.7-6.3-2.4-16.7-3.2-21.8-1.8l-2.8.8 2.6 4.4c4.8 8.1 4.3 20-1.1 26.5-2.1 2.5-3.3 3-6.8 3-2.3 0-5.1-.7-6.2-1.5-5.8-4-7.6-17.7-3.5-26.9 1.1-2.7 2.1-5 2.1-5.1 0-.1-1.5-.9-3.2-1.6-4.4-1.8-14.4-1.6-19.1.4l-3.7 1.6 3.1 4.5c3.5 4.9 4.6 11.2 3.4 19.2-1.3 8.6-9.1 13.2-15.3 8.8-5.9-4.1-8.7-16.5-5.8-25.5 1-3 1.4-5.5.8-6.1-.4-.4-4.2-1.5-8.3-2.4-12.8-2.7-22.7-.3-31.2 7.5-4.6 4.3-7.4 5.1-8.3 2.6-.7-1.7-2.4 3.8 25.5-81.3l22.2-67.2h3.5c3.2 0 3.5.2 2.8 2.2z"/><path d="M267.9 342.6c-.7 1.6-1.3 5.7-1.3 9-.1 4.8.3 6.7 1.9 8.4 1.8 2 2 2 3.2.4 3.2-4.3 2.4-16.4-1.2-19.4-1.2-1-1.7-.7-2.6 1.6zM230 343.9c-2.5 4.7 0 18.1 3.3 18.1 2 0 2.7-2.2 2.7-8.3 0-3.4-.7-6.3-2-8.5-2.3-3.7-2.6-3.8-4-1.3z"/></svg>
```

Comment:
We're missing a way to tell safari to use this:

https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html

`<link rel="mask-icon" ...`

(I'm not sure how important a use-case this is, if you think we have Safari users who pin webcompat.com as a pinned tab, we should add the `<link>`, otherwise maybe we should just delete the svg entirely).

Reply:
Great catch. Thanks!